service/src/main/resources/api/service_openapi.yaml

```diff
@@ -2338,6 +2338,8 @@ components:
     FeatureSetCloneInfo:
       type: object
       properties:
+        studyId:
```

Comment:
nit: `destinationStudyId` to make this clearer.

Also can you add a comment noting that this is optional, and if not provided we'll clone within the same study.

Reply:
was going back and forth on the name and description. by default it is optional
renamed, and added a comment (not description since it is not allowed if we ref to a definition)